.gitignore

```diff
@@ -5,6 +5,13 @@ yarn.lock
 _site
 package-lock.json
 
+# Project dependencies
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
```

Comment:
not sure if this intended to be helpful for future reference, however this link is broken ;-)

Reply:
As you mentioned it does not have any use, so I deleted it.
Thanks for noticing.